README.md

```diff
@@ -51,16 +53,16 @@ _Note_: You will want to include this either at the beginning of your script inc
 
 ### getValue
 
-Takes a key of the configuration value you wish to return.
+This function takes a parameter of the key of the configuration value you wish to return, _e.g._ `"title"`.
 
-If the config hasn't been set, you are in an environment that doesn't exist, or pass in a key that doesn't exist you will get a console.error describing what went wrong.
+You will get a console.error describing what went wrong if the config hasn't been set, you are in an environment that doesn't exist, or you pass in a key that doesn't exist.
```

Comment:
console error is probably the way we would say this in a sentence, despite this being technically correct.

Reply:
Then how about "You will get an error in the console"?